### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pyrogram'

### DISCUSSION THREAD

**Discussion by ghost** [Dineshkarthik/telegram_media_downloader#217] [Q&A] [answered]
Hi. I am receiving errors when running python3 media_downloader.py 

> Traceback (most recent call last):
  File "C:\WINDOWS\system32\telegram_media_downloader\telegram_media_downloader\media_downloader.py", line 8, in <module>
    import pyrogram
ModuleNotFoundError: No module named 'pyrogram'

Can you please help? Thank you!

**Comment by tissole** [comment, marked as answer]
Pyrogram is not installed in your system. Open powershell type `cd "C:\WINDOWS\system32\telegram_media_downloader\telegram_media_downloader"` click enter and then `pip install -r requirements.txt`

**Comment by SkBashaOp** [comment]
PS C:\Users\hussa\Downloads\telegram_media_downloader-2.0.2> python3 media_downloader.py
Traceback (most recent call last):
  File "C:\Users\hussa\Downloads\telegram_media_downloader-2.0.2\media_downloader.py", line 7, in <module>
    import pyrogram
ModuleNotFoundError: No module named 'pyrogram'
PS C:\Users\hussa\Downloads\telegram_media_downloader-2.0.2> 
Same error used cmd run as admin powershell run as admin in vscode also tried help me try to do gmeet
![image](https://github.com/Dineshkarthik/telegram_media_downloader/assets/73814469/08260037-4cb1-4bd9-8a8c-1c6a7b570029)
